Doc/library/typing.rst

```diff
@@ -1059,7 +1059,9 @@ These are not used in annotations. They are building blocks for creating generic
       given call and should only be used to annotate ``*args``.  ``P.kwargs``
       represents the mapping of keyword parameters to their values in a given call,
       and should be only be used to annotate ``**kwargs`` or ``**kwds``.  Both
```

Comment:
```suggestion
      and should be only be used to annotate ``**kwargs``.  Both
```
(It should be clear that the *name* of that argument is irrelevant, only the `**` prefix matters.)